config/minimalist-templates/popups_templates.yaml

```diff
@@ -0,0 +1,531 @@
+                    #     - height: 25px
+                    #     - width: 45px
+                    #     - font-size: 11px
+                    #     - line-height: 25px
+                    #     - color: white
+                  custom_fields:
+                    item1:
+                      card:
+                        type: "custom:slider-card"
+                        entity: "[[[ return entity.entity_id ]]]"
+                        radius: 20px
+                        rotate: 270
+                        containerHeight: 320px
+                        width: 250px
+                        height: 100px
+                        top: 100px
+                        left: 17%
+                        thumbHorizontalPadding: 0px
+                        thumbWidth: 0px
+                        mainSliderColor: rgba(var(--color-theme),0.1)
+                        mainSliderColorOff: rgba(var(--color-theme),0.1)
+                        secondarySliderColor: var(--color-theme)
+                        secondarySliderColorOff: var(--color-theme)
+                        card_mod:
+                          style: |
+                            ha-card {
+                              background: none !important;
+                              box-shadow: none !important;
+                            }
+                            .off {
+                              border: 1px solid rgba(var(--color-theme),0.3) !important;
+                            }
+                            .on {
+                              border: 1px solid rgba(var(--color-theme),0.3) !important;
+                            }
+                    # item2: >
+                    #   [[[
+                    #     var bri = states[entity.entity_id].attributes.brightness;
+                    #     return Math.round(bri ? (bri/256*100) : '0') + '%';
+                    #   ]]]
+                #####
+                color:
+                  type: custom:light-entity-card
+                  entity: "[[[ return entity.entity_id ]]]"
+                  persist_features: true
+                  brightness: false
+                  color_temp: false
+                  white_value: false
+                  hide_header: true
+                  effects_list: false
+                  card_mod:
+                    style:
+                      ".control":
+                        $: |
+                          #interactionLayer{ z-index: 1; filter: opacity(0.6);}
+                          #backgroundLayer{
+                            filter: blur(20px);
+                          }
+                      .: |
+                        ha-card {
+                          background: none !important;
+                          box-shadow: none !important;
+                          padding: 10px 0 10px 0 !important;
+                        }
+          item3:
+            card:
+              type: custom:button-card
+              template: popup_list_items
```

Comment:
```suggestion
              styles:
                grid:
                  - grid-template-areas: >
                      [[[
                            var areas = [];
                            var brightness_modes = ["brightness", "hs", "color_temp", "rgb", "rgbw"];
                            var color_modes = ["hs", "rgb", "rgbw"];
                            var color_temp_modes = ["color_temp"];
                            if (entity.attributes.supported_color_modes.some(r => brightness_modes.includes(r))) {
                                areas.push("item1");
                            }
                            if (entity.attributes.supported_color_modes.some(r => color_modes.includes(r))) {
                                areas.push("item2");
                            }
                            if (entity.attributes.supported_color_modes.some(r => color_temp_modes.includes(r))) {
                                areas.push("item3");
                            }
                            return "\"" + areas.join(" ") + "\"";
                      ]]]
                  - grid-template-columns: >
                      [[[
                            var columns = [];
                            var brightness_modes = ["brightness", "hs", "color_temp", "rgb", "rgbw"];
                            var color_modes = ["hs", "rgb", "rgbw"];
                            var color_temp_modes = ["color_temp"];
                            if (entity.attributes.supported_color_modes.some(r => brightness_modes.includes(r))) {
                                columns.push("1fr");
                            }
                            if (entity.attributes.supported_color_modes.some(r => color_modes.includes(r))) {
                                columns.push("1fr");
                            }
                            if (entity.attributes.supported_color_modes.some(r => color_temp_modes.includes(r))) {
                                columns.push("1fr");
                            }
                            return columns.join(" ");
                      ]]]
                  - grid-template-rows: min-content
                  - column-gap: 0px
                card:
                  - background: none
                  - box-shadow: none
                  - padding: 0px
```
I've ditched the popup_list_items and created my own grid layout. The layout is calculated on how many features are supported by the entity. I don't know why, but you also need a 'display: none' on the items below. (see the next change) I would think that not specifying items here in the grid layout removes them from the layout. But there just appended to the grid. So the solution is a bit verbose, but it works quite well.